[2025-08-05T19:30:23.294+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipeline_medallon.cargar_a_bulk manual__2025-08-05T19:29:40.968196+00:00 [queued]>
[2025-08-05T19:30:23.327+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipeline_medallon.cargar_a_bulk manual__2025-08-05T19:29:40.968196+00:00 [queued]>
[2025-08-05T19:30:23.328+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-08-05T19:30:23.373+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): cargar_a_bulk> on 2025-08-05 19:29:40.968196+00:00
[2025-08-05T19:30:23.378+0000] {standard_task_runner.py:60} INFO - Started process 170 to run task
[2025-08-05T19:30:23.382+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'pipeline_medallon', 'cargar_a_bulk', 'manual__2025-08-05T19:29:40.968196+00:00', '--job-id', '268', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_medallon_dag.py', '--cfg-path', '/tmp/tmpn1ey1b02']
[2025-08-05T19:30:23.384+0000] {standard_task_runner.py:88} INFO - Job 268: Subtask cargar_a_bulk
[2025-08-05T19:30:23.469+0000] {task_command.py:423} INFO - Running <TaskInstance: pipeline_medallon.cargar_a_bulk manual__2025-08-05T19:29:40.968196+00:00 [running]> on host 70eaa744a05c
[2025-08-05T19:30:23.666+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mateo.biasotto' AIRFLOW_CTX_DAG_ID='pipeline_medallon' AIRFLOW_CTX_TASK_ID='cargar_a_bulk' AIRFLOW_CTX_EXECUTION_DATE='2025-08-05T19:29:40.968196+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-05T19:29:40.968196+00:00'
[2025-08-05T19:30:23.679+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-08-05T19:30:23.687+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /opt/airflow/scripts/minio_04_carga_a_transito.py']
[2025-08-05T19:30:23.728+0000] {subprocess.py:86} INFO - Output:
[2025-08-05T19:30:25.957+0000] {subprocess.py:93} INFO - Conectando a MinIO en minio:9000...
[2025-08-05T19:30:25.958+0000] {subprocess.py:93} INFO - Tablas bulk limpiadas correctamente antes de la carga.
[2025-08-05T19:30:25.959+0000] {subprocess.py:93} INFO - Cargando dim_cliente.parquet (4919 filas) a bulk_dim_cliente...
[2025-08-05T19:30:25.959+0000] {subprocess.py:93} INFO -    ðŸ”¹ bulk_dim_cliente: 4919 â†’ 801 filas tras limpiar nulos y duplicados en customer_id.
[2025-08-05T19:30:25.960+0000] {subprocess.py:93} INFO - 801 filas cargadas en bulk_dim_cliente
[2025-08-05T19:30:25.961+0000] {subprocess.py:93} INFO - Cargando dim_producto.parquet (1894 filas) a bulk_dim_producto...
[2025-08-05T19:30:25.961+0000] {subprocess.py:93} INFO -    ðŸ”¹ bulk_dim_producto: 1894 â†’ 1862 filas tras limpiar nulos y duplicados en product_id.
[2025-08-05T19:30:25.962+0000] {subprocess.py:93} INFO - 1862 filas cargadas en bulk_dim_producto
[2025-08-05T19:30:25.962+0000] {subprocess.py:93} INFO - Cargando fact_ventas.parquet (10004 filas) a bulk_fact_ventas...
[2025-08-05T19:30:25.963+0000] {subprocess.py:93} INFO -    ðŸ”¹ bulk_fact_ventas: 10004 â†’ 9986 filas tras limpiar nulos y duplicados en (order_id, product_id).
[2025-08-05T19:30:25.964+0000] {subprocess.py:93} INFO - 9986 filas cargadas en bulk_fact_ventas
[2025-08-05T19:30:26.102+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-08-05T19:30:26.166+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=pipeline_medallon, task_id=cargar_a_bulk, execution_date=20250805T192940, start_date=20250805T193023, end_date=20250805T193026
[2025-08-05T19:30:26.253+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-08-05T19:30:26.390+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
